js/deleteTask.js

```diff
@@ -0,0 +1,6 @@
+export default function removeTodo(element, taskList) {
+  const index = taskList.findIndex((item) => item.id == element.id);
```

Comment:
always use `===` for comparison.

```suggestion
  const index = taskList.findIndex((item) => item.id === element.id);
```